### PR TITLE
Add recipe for jemalloc

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

**Comment by scopatz** [comment]
Please enable CircleCI on your fork of staged recipes, see http://conda-forge.github.io/docs/ciservices.html#circleci-linux

**Comment by xhochy** [comment]
CircleCI enabled and all tests pass now.

File: recipes/jemalloc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+about:
+  home: http://jemalloc.net
+  dev_url: https://github.com/jemalloc/jemalloc
+  license: BSD 2-Clause
+  license_file: COPYING
+  summary: 'general purpose malloc(3) implementation'
+
+extra:
+  recipe-maintainers:
+    - wesm
```
Comment (scopatz) [inline recipes/jemalloc/meta.yaml]: @wesm are you OK with being added here?

Comment (wesm) [inline recipes/jemalloc/meta.yaml]: yes

**Pull request merged**